sage-android/Sage/app/src/main/java/blueprint/com/sage/checkIn/fragments/CreateCheckInFragment.java

```diff
@@ -240,11 +241,13 @@ private boolean areFieldsEmpty(String... params) {
     }
 
     private void resetCheckIn() {
+        SessionUtils.updateRequestCount(getActivity(), 1, R.string.admin_check_in_requests);
```

Comment:
This should only happen if the admin is making a request to a school that they have to approve of